starter-code/routes/movies.routes.js

```diff
@@ -0,0 +1,50 @@
+const express = require("express");
+const router = express.Router();
+require("dotenv").config();
```

Comment:
dotenv package was already required and handled in app.js so it doesn't need to be executed again.